x/protocolpool/types/genesis.go

```diff
@@ -6,22 +6,27 @@ import (
 	errorsmod "cosmossdk.io/errors"
 	"cosmossdk.io/math"
 
+	sdk "github.com/cosmos/cosmos-sdk/types"
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 )
 
 func NewGenesisState(cf []*ContinuousFund, budget []*Budget) *GenesisState {
 	return &GenesisState{
 		ContinuousFund: cf,
 		Budget:         budget,
-		LastBalance:    math.ZeroInt(),
+		LastBalance:    DistributionAmount{Amount: sdk.NewCoins()},
 		Distributions:  []*Distribution{},
+		Params:         &Params{},
 	}
 }
 
 func DefaultGenesisState() *GenesisState {
 	return &GenesisState{
 		ContinuousFund: []*ContinuousFund{},
 		Budget:         []*Budget{},
+		Params: &Params{
+			EnabledDistributionDenoms: []string{"stake"},
```

Comment:
Blocker: We need a way to make this configurable by chains. 

Not sure if `sdk.DefaultBondDenom` is still used but you can make this a constructor param and pass it from the app module

Reply:
They can change this in the genesis file, but I'll change it to sdk.DefaultBondDenom, not sure what I thought about when I put there the string